### DISCUSSION TITLE
Prevent default styles from overwriting my CSS

### DISCUSSION THREAD

**Discussion by elgeneee** [heroui-inc/heroui#397] [Help] [answered]
I am using TailwindCSS to style all my components, I am only interested in using NextUI input and table component due to the looks of it. However, after following the guide from Get Started. All my texts are changed, not following the styles input by tailwind. Would appreciate any help if I am only interested in using some of the components provided by NextUI without altering my existing styles.

**Comment by jrgarciadev** [comment, marked as answer]
Hey @elgeneee you can disable the NextUI's base styles using this prop `disableBaseline` in the `NextUIProvider`
https://nextui.org/docs/guide/getting-started#nextuiprovider-props

e.g

```js
// 1. import `NextUIProvider` component
import { NextUIProvider } from '@nextui-org/react';

function MyApp({ Component, pageProps }) {
  return (
    // 2. Use at the root of your app
    <NextUIProvider disableBaseline>
      <Component {...pageProps} />
    </NextUIProvider>
  );
}

export default MyApp;
```

Hope it helps you 🙌🏻

**Reply by elgeneee** [reply]
Thank you!

**Comment by ymittal** [comment]
How do I do this in v2? It doesn't seem like v2 has a `disableBaseline` prop

**Comment by MaxLikesCode** [comment]
What is the solution for v2?

**Comment by sal-the-dev** [comment]
Bumping this. Has anyone found a solution yet?

**Comment by dclstn** [comment]
This no longer works in v2, version bumps shouldn't result in loss of functionality
